PackageSourceMappingExample/README.md

```diff
@@ -1,9 +1,322 @@
-# Package Source Mapping example
+# Package Source Mapping sample
 
-This is an example demonstrating the [Package Source Mapping feature](https://github.com/NuGet/Home/blob/dev/proposed/2021/PackageNamespaces.md) (initially proposed as "Package Namespaces").
+This folder is an example demonstrating the [Package Source Mapping feature](https://devblogs.microsoft.com/nuget/introducing-package-source-mapping/) (initially proposed as "Package Namespaces").
 
-  * [Visual Studio 2022 preview 4 and later](https://visualstudio.microsoft.com/vs/preview/)
-  * [nuget.exe 6.0.0-preview.4 and later](https://www.nuget.org/downloads)
-  * [.NET SDK 6.0.100-rc.1 and later](https://dotnet.microsoft.com/download/dotnet/6.0)
+In addition to the `nuget.config` file in this folder, you can also check out how we set up our own [`nuGet.config` for the NuGet.Client repo](https://github.com/NuGet/NuGet.Client/blob/dev-kartheekp-ms-dogfood-namespaces/NuGet.Config) to use this feature internally.
+
+## Package Source Mapping tooling
+
+This feature is compatible with the following tools:
+
+*   [Visual Studio 2022 preview 4][1] and later
+*   [nuget.exe 6.0.0-preview.4][5] and later
+*   [.NET SDK 6.0.100-rc.1][6] and later
+
+Older tooling will ignore the Package Source Mapping configuration. To use this feature, ensure all your build environments use compatible tooling versions.
+
+Package Source Mappings will apply to all project types - including .NET Framework - as long as compatible tooling is used for build and restore.
 
 Here's an [example](https://github.com/NuGet/NuGet.Client/blob/dev-kartheekp-ms-dogfood-namespaces/NuGet.Config) of how the NuGet.Config for the NuGet.Client repo looks like in our dogfooding efforts.
+
+## Package Source Mapping rules

```

Comment:
I have some feedback on these rules, but given that there's a lot of copying, I'm wondering if it's best to link to the blog itself?

Reply:
Samples are not the right place to write documentation. It should be on docs.microsoft.com, or maybe the blog post. Samples should be minimal examples of how a feature works. Consider azure samples, win32 samples, winui/xaml samples, pretty much any other sample repo.

Reply:
Removed the docs duplication to keep the sample more minimal.